app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,62 @@
+@media (min-width: 768px) {
+  .omb_row-sm-offset-3 div:first-child[class*="col-"] {
+    margin-left: 25%;
+  }
+}
+
+.omb_login .omb_authTitle {
```

Comment:
Selector `omb_login` should be written in lowercase with hyphens<br>Selector `omb_authTitle` should be written in lowercase with hyphens